PortalTests/View/UIKit/Renderers/LabelRendererSpec.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  LabelRendererSpec.swift
+//  PortalTests
+//
+//  Created by Pablo Giorgi on 8/28/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import Portal
+
+class LabelRendererSpec: QuickSpec {
+    override func spec() {
+
+        var layoutEngine: LayoutEngine!
+
+        beforeEach {
+            layoutEngine = YogaLayoutEngine()
+        }
+
+        describe(".apply(changeSet: TextFieldChangeSet) -> Result") {
+
+            var changeSet: LabelChangeSet<String>!
+
+            beforeEach {
+
+                let labelProperties: LabelProperties<String> = properties(
+                    text: "Hello World before layout",
+                    textAfterLayout: "Hello World"
+                )
+
+                let labelStyle = labelStyleSheet { base, label in
+                    label.textColor = .red
+                    label.textFont = Font(name: "Helvetica")
+                    label.textSize = 12
+                    label.textAligment = .center
+                    label.adjustToFitWidth = true
+                    label.numberOfLines = 0
+                    label.minimumScaleFactor = 1.0
+                }
+
+                changeSet = LabelChangeSet.fullChangeSet(
+                    properties: labelProperties,
+                    styleSheet: labelStyle,
+                    layout: layout()
+                )
+                
+            }
+
+            context("when the change set contains label property changes") {
+
+                it("applies 'text' property changes") {
```

Comment:
This should be refactored to

```swift
let result = label.apply(changeSet: changeSet, layoutEngine: layoutEngine)
result.afterLayoutTask?()
expect(label.text).to(equal("Hello World"))
```

Reply:
Nice, that's what I wanted.

I thought `afterLayoutTask` was a closure executed after layout.

Reply:
Don't you should check `text` property? (before layout)